mkosi/distributions/gentoo.py

```diff
@@ -271,21 +187,30 @@ def set_useflags(self) -> None:
                 """
             )
         )
+        if state.config.make_initrd:
+            package_use.joinpath("minimal").write_text(
+                dedent(
+                    """\
+                    # MKOSI
+                    */* minimal
+                    """
+                )
+            )
```

Comment:
We should do the `minimal` stuff by default, not conditional on the value of `config.make_initrd`

Reply:
right, that's what the `flock` and `cache_isclean` were for. i can put them back.